frontend/src/components/Scratch/ScratchToolbar.tsx

```diff
@@ -115,7 +115,7 @@ function ScratchName({ name, onChange }: { name: string, onChange?: (name: strin
         />
     } else {
         return <div
-            className={classNames(styles.name, { [styles.editable]: !!onChange })}
+            className={classNames(styles.name, styles.scratchName, { [styles.editable]: !!onChange })}
```

Comment:
Would it make more sense to add these css attributes to the `.name` class instead? and then make `cursor: text;` the default rather than only when `&.editable` ?